bayesian.py

```diff
@@ -260,26 +260,26 @@ def getSign(p):
                     variable = queryAssignment[i][1:]
                     result *= enumerationAsk(states,bayesNet,variable,typeRes, events)
                     count+=1
-            print '%.7f'%result    
-                
+            print(('%.7f'%result).rstrip('0'))
+
         else: # only one query
             evidence = ev.split(',')
-        
+
             typeRes = getSign(assigment)
             variable = assigment[1:]
-            
+
             #create dictionary of events
             signs = []
             for e in evidence:
                 sign = getSign(e)
                 value = e[1:]
                 events.update({value:sign})
-            
+
             result =  enumerationAsk(states,bayesNet,variable,typeRes, events)
-            print '%.7f'%result
+            print(('%.7f'%result).rstrip('0'))
 
     else:
         typeRes = getSign(query)
         variable = query[1:]
         result =  enumerationAsk(states,bayesNet,variable,typeRes, {})
-        print '%.7f'%result
\ No newline at end of file
+        print(('%.7f'%result).rstrip('0'))
```

Comment:
@lillyann9 did you check your output against the testcases? Your format is slightly off.